File: tools/dbench/answers/python_sonnet/s2/f2_adults.py
names = []
with open("users.txt") as f:
    for line in f:
        p = line.split()
        if len(p) == 3:
            try:
                if int(p[2]) > 25:
                    names.append(p[1])
            except ValueError:
                pass
print(" ".join(names))

File: tools/dbench/answers/rust_sonnet/s3/f2_adults.rs
use std::fs;

fn main() {
    let text = fs::read_to_string("users.txt").unwrap_or_default();
    let names: Vec<&str> = text.lines()
        .filter_map(|line| {
            let parts: Vec<&str> = line.split_whitespace().collect();
            if parts.len() == 3 {
                if let Ok(age) = parts[2].parse::<i64>() {
                    if age > 25 { return Some(parts[1]); }
                }
            }
            None
        })
        .collect();
    println!("{}", names.join(" "));
}
